crypto/sha/keccak1600.c

```diff
@@ -321,6 +343,149 @@ void KeccakF1600(uint64_t A[5][5])
     }
 }
 
+#elif defined(KECCAK_1X_ALT)
+/*
+ * This is variant of above KECCAK_1X that presses temporary storage
+ * even further, but at cost of more write references to A[5][5].
+ * It's less suitable if A[5][5] is memory bound, but better if it's
```

Comment:
I don't know what "presses temporary storage" means.  Uses more ?  Stack?

By "A[5][5]" you mean the whole array, not just that one subscript?  Perhaps "array A[][]" is a better comment?